scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/SubmitWindow.java

```diff
@@ -1274,15 +1282,18 @@ private void build() {
         setEnabledStartAtPart(false);
 
         this.window = new Window();
-        this.window.setTitle("Submit a new job");
+        this.window.setTitle(isResubmit ? "Re-submit job " + jobId : "Submit a new job");
         this.window.setShowMinimizeButton(false);
         this.window.setIsModal(true);
         this.window.setShowModalMask(true);
         this.window.addItem(rootPage);
-        this.window.setWidth(this.widthWindows);
-        this.window.setHeight(this.heightWindows);
+        this.window.setWidth(WINDOW_WIDTH);
+        this.window.setHeight(WINDOW_HEIGHT);
         this.window.centerInPage();
         this.window.setCanDragResize(true);
+        if (isResubmit) {
+            uploadJobXMLForResubmit();
+        }
     }
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Rename "variables" which hides the field declared at line 237. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AHiddenFieldCheck)